iproute: include netem tools

Ralith opened the pull request:
###### Motivation for this change

Fixes errors like 
> No distribution data for normal (/nix/store/djgbl8rvjs2h3jmq65182yahchws8s35-iproute2-5.2.0/lib/tc//normal.dist: No such file or directory)

from `tc` when using netem. It's unclear why these were removed, but the removal seems to be dated and netem seems to work fine with this fix.

###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

primeos reviewed (changes requested):
Unfortunately `HOSTCC` is still required as the compiled binaries need to be executed to generate the `*.dist` files (containing the distributions). I.e. in the current form this would unfortunately still break cross-compilation. But the `.dist`-files are indeed installed now.

I see two possibilities:
1) Only build `netem` when not cross-compiling
2) Fix the cross-compiling by making the host compiler available via `HOSTCC` (should be fine as only the resulting `.dist`-files are installed, not the binaries - seems like the preferable solution)

FRidh commented:
ping @Ralith

Ralith commented:
I don't have much grasp of cross-compilation in nixpkgs or the tc build system, and limited time for this issue. I'm happy to make changes but I need more to go on than "Fix the cross-compiling by making the host compiler available via `HOSTCC`".

primeos committed:
iproute: Simplify and improve the expression

primeos committed:
iproute: Build the netem tools

They will be installed now and we can provide $HOSTCC for
cross-compilation.

New files:
+lib/tc/experimental.dist
+lib/tc/normal.dist
+lib/tc/pareto.dist
+lib/tc/paretonormal.dist

Note: The distributions are generated in a reproducible way.

Co-Authored-By: Benjamin Saunders <ben.e.saunders@gmail.com>

primeos reviewed (approved):
I've had a look and updated the PR.
I verified with `diffoscope` that this only adds new files (with the exception that `$out` obviously changes) and tested cross-compilation with `pkgsCross.aarch64-multiplatform-musl.iproute`. It should be fine now :)
